### ISSUE TITLE
Old reports with different JSON structure cannot be imported 

### ISSUE THREAD

**Issue by mhansma96** [w3c/wcag-em-report-tool#511]
Old reports have a different JSON format and within the import-fuction all kinds of things will give errors because some keys don't exist in exports from the old tool

**Comment by mhansma96** [comment]
I've made some changes that should make it possible for old reports to be imported. But I do lack some old reports or older knowledge of potential scenario's that can fail. We should try to test a few more reports on their compatibility with the new tool after an import. Preferably reports from the old tool. @hidde @nitedog

**Comment by hidde** [comment]
Great, thanks for those changes, Michel! 

> We should try to test a few more reports on their compatibility with the new tool after an import. Preferably reports from the old tool.

Agreed. @nitedog do you have any old reports we could use for testing this?

**Comment by nitedog** [comment]
No, unfortunately not anymore. Maybe EricV can get some anonymized reports from ACC?
